altair/utils/schemapi.py

```diff
@@ -181,19 +240,53 @@ def __str__(self):
             for val in schema_path[:-1]
             if val not in (0, "properties", "additionalProperties", "patternProperties")
         )
-        message = self.message
-        if self._additional_errors:
-            message += "\n        " + "\n        ".join(
-                [e.message for e in self._additional_errors]
-            )
-        return """Invalid specification
```

Comment:
I moved this down because I don't think there will ever be multiple lines in the error message if it contains an unknown parmater name which is what the next if condition tests.